flow-decomposition/src/main/java/com/powsybl/flow_decomposition/FlowDecompositionParameters.java

```diff
@@ -15,6 +15,10 @@
  * @author Hugo Schindler {@literal <hugo.schindler at rte-france.com>}
  */
 public class FlowDecompositionParameters {
+    public enum XnecSelectionStrategy {
+        ONLY_INTERCONNECTIONS,
+        ZONE_TO_ZONE_PTDF_CRITERIA,
```

Comment:
should this actually be called "INTERCONNNECTION_OR_FULFILLS_5PC_PTDF_CRITERIA" ?

Reply:
Based on the [article 6.6.a (page 9)](https://eepublicdownloads.entsoe.eu/clean-documents/Network%20codes%20documents/Implementation/ccr/methodologies/core_ccr_tsos_regional_design_of_id_fb_ccm_june18_final.pdf)

> A cross-zonal network element is always considered as significantly influenced. The other CNECs shall have a maximum zone-to-zone Power Transfer Distribution Factor (𝑃𝑇𝐷𝐹), as described in Article 13, higher than a common threshold of 5 percent

But I agree that the name of this Strategy is not ideal. I'll change it to `INTERCONNECTION_OR_ZONE_TO_ZONE_PTDF_GT_5PC`